runners/apex/src/main/java/org/apache/beam/runners/apex/translation/FlattenPCollectionTranslator.java

```diff
@@ -16,17 +16,17 @@
  * limitations under the License.
  */
 
-package org.apache.beam.runners.apex.translators;
+package org.apache.beam.runners.apex.translation;
 
 import com.google.common.collect.Lists;
 
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
-import org.apache.beam.runners.apex.translators.functions.ApexFlattenOperator;
-import org.apache.beam.runners.apex.translators.io.ApexReadUnboundedInputOperator;
-import org.apache.beam.runners.apex.translators.io.ValuesSource;
+import org.apache.beam.runners.apex.translation.functions.ApexFlattenOperator;
+import org.apache.beam.runners.apex.translation.io.ApexReadUnboundedInputOperator;
```

Comment:
Do the `functions` and `io` package add anything? It seems like it forces some things to be public that don't need to be. Or maybe if someone really declares a dependency of `beam-runners-apex` then having access to those would be useful? I almost think all the classes could reside in one package. Users mostly just need the `ApexRunner` and it isn't that many classes. Just vague questions; I trust your understanding and judgment.